### DISCUSSION TITLE
vscode icon of github copilot is now showing

### DISCUSSION THREAD

**Discussion by Tushar1012** [community/community#17463] [Copilot Conversations] [answered]
vscode icon of GitHub copilot is now showing
 don't know what is happening i couldn't use GitHub copilot.

**Comment by davecheney** [comment, marked as answer]
Your copilot extension is out of date. Please upgrade.

**Reply by XeiDaMoKa** [reply]
doesnt work for me , nonof the copilot extensions show the icon , but it worksin ctrl enter

**Reply by davecheney** [reply]
Which version of vscode are you using? Which version of the copilot extension are you using?

**Reply by XeiDaMoKa** [reply]
i believe it was the 1.67 when i downloaded from the vsc website like a week ago, didnt work , then i tried the windows store version which is what i currently have at 1.68 and the copilot is the normal one v1.28.6134 , and i tried the nightly v1.27.6124 still no icon , i gave permissions for vsc in the windows defender as i saw someone online fixing it and downloaded again with no antivirus , accepted all popups in vsc about it but no icons appear, but gives me sugestions while coding no problem , but i wanted to try to conversion tool

**Comment by awgn** [comment]
HI there,

I'm running copilot v1.30.6165 and vscode 1.68.1. 

Copilot is working but for some reason the icon is not shown in the bar. I tried uninstalling and installing it again without success. Any suggestion?
